ios/deploy_ios.sh

```diff
@@ -1,27 +1,19 @@
-#!/bin/sh -e
+#!/bin/bash
+set -eu
 
-# autobuild_2_build: actual build process
-
-
-####################
-###  PARAMETERS  ###
-####################
-
-
-source "$(dirname "${BASH_SOURCE[0]}")/../ensure_THIS_JAMULUS_PROJECT_PATH.sh"
-
-###################
-###  PROCEDURE  ###
-###################
-
-cd "${THIS_JAMULUS_PROJECT_PATH}"
+## Builds an ipa file for iOS. Should be run from the repo-root
 
+# Create Xcode file and build
 qmake -spec macx-xcode Jamulus.pro
 /usr/bin/xcodebuild -project Jamulus.xcodeproj -scheme Jamulus -configuration Release clean archive -archivePath "build/Jamulus.xcarchive" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
-# Make a deploy folder
-mkdir deploy
+
 # Generate ipa by copying the .app file from the xcarchive file
```

Comment:
`xcarchive file` looks like a directory.

Reply:
I don't know anything about that part, but based on the `cp` invocation, it must be a directory, so I've adjusted the comment as well (cc @ann0see).

Reply:
On macOS it displays as a file although you can cd into it. It’s similar to .app files. They show up as file, but are essentially folders

Reply:
A truely broken design...